charts/cadence/custom_values.yaml

```diff
@@ -0,0 +1,26 @@
+# -- Define this spec locally and pass it to below command if you want to setup Grafana/Prometheus along with serviceMonitor:
```

Comment:
As discussed offline let's create examples/grafana folder and move this file there. 
Add a readme file there to explain how to setup grafana dashboards. Do not change the README.md at top level